src/scf-release/src/acceptance-tests-brain/test-scripts/017_syslog_forwarding_test.rb

```diff
@@ -119,6 +119,7 @@ def emit_log_entries
     socat_args = "/usr/bin/socat #{SCF_LOG_PROTOCOL.upcase}-LISTEN:#{SCF_LOG_PORT},fork 'EXEC:/usr/bin/logger --socket-errors=off --stderr --tag \"\"'"
     cmd = %W(
         kubectl run #{$LOG_SERVICE_NAME}
+            --generator=run-pod/v1
```

Comment:
There is an open PR for this bit already https://github.com/SUSE/scf/pull/2187/files. Since we skipped brain tests on this PR, how about removing it from here?